test/Microsoft.AspNet.Mvc.Core.Test/Filters/AuthorizeFilterTestsBase.cs

```diff
@@ -12,7 +12,7 @@
 
 namespace Microsoft.AspNet.Mvc.Core.Test
 {
-    public class AuthorizeAttributeTestsBase
+    public class AuthorizeFilterTestsBase
     {
         protected AuthorizationContext GetAuthorizationContext(Action<ServiceCollection> registerServices, bool anonymous = false)
```

Comment:
This is wierd and only used in one place. I know it's not a mess that you made but you can just move this method to the other class and delete this?